.env

```diff
@@ -0,0 +1,12 @@
+MYSQLDB_DATABASE=book_app
```

Comment:
add `.env` to `.gitignore` and remove from remote repo  
Better add `.env.template` without values